src/TodoInput.js

```diff
@@ -0,0 +1,21 @@
+import {KEY} from './constants.js';
+
+export default function TodoInput($todoInput, addItem) {
+  if (!$todoInput) {
+    throw new Error('ERROR: Invalid object');
+  }
+
+  this.$todoInput = $todoInput;
+
+  this.$todoInput.addEventListener('keyup', (e) => {
+    if (e.key === KEY.ENTER) {
+      if (!this.$todoInput.value) {
+        alert('할일을 입력하세요!');
+        return;
+      }
+
+      addItem(this.$todoInput.value);
+      this.$todoInput.value = '';
+    }
+  });
```

Comment:
```suggestion
  this.$todoInput.addEventListener('keyup', ({ key, target }) => {
    if (key === KEY.ENTER) {
      if (!target.value) {
        alert('할일을 입력하세요!');
        return;
      }

      addItem(target.value);
      target.value = '';
    }
  });
```
이 이벤트 메소드에서는 굳이 this를 사용할 필요는 없을 것 같아요!

Reply:
핫 그렇네요! 꼼꼼히 봐주셔서 감사드립니닷👍